SConstruct

```diff
@@ -1479,16 +1459,23 @@ addInstallActions = ('install' in COMMAND_LINE_TARGETS or
 
 # Directories where things will be staged for package creation. These
 # variables should always be used by the Install(...) targets
-if env['stage_dir']:
-    instRoot = pjoin(os.getcwd(), env['stage_dir'],
-                     stripDrive(env['prefix']).strip('/\\'))
-    if env['python_prefix']:
-        env['python_prefix'] = pjoin(os.getcwd(), env['stage_dir'],
-                       stripDrive(env['python_prefix']).strip('/\\'))
+if env["stage_dir"]:
+    stage_prefix = Path(env["prefix"])
+    # Strip the root off the prefix if it's absolute
+    if stage_prefix.is_absolute():
+        stage_prefix = Path(*stage_prefix.parts[1:])
+
+    instRoot = Path.cwd().joinpath(env["stage_dir"], stage_prefix)
```

Comment:
Renaming this here would require renaming it in other places in this file, which I was trying to avoid (to keep this PR focused on changing `buildutils.py` and the minimum set of changes to keep things working). A few things slipped through that don't strictly follow that, like adding the `logger` in a few places, but that was my general thought.